CHANGELOG.md

```diff
@@ -1,6 +1,17 @@
 Changelog
 =========
 
+## vNext
+
+### Features
+
+- "Delegation" screens UI:
```

Comment:
@DeeJayElly why we have this entry in CHANGELOG, should be just your `Chore`?

Reply:
Fixed @tomothespian

Reply:
thx 🎉